crates/rpc-types/src/eth/transaction/error.rs

```diff
@@ -28,6 +28,9 @@ pub enum ConversionError {
     /// Missing `chainId` field for EIP-1559 transaction.
     #[error("missing `chainId` field for EIP-155 transaction")]
     MissingChainId,
+    /// Missing `chainId` field for EIP-1559 transaction.
+    #[error("missing `blobVersionedHashes` field for EIP-4844 transaction")]
+    MissingBlobVersionedHashes,
```

Comment:
Can we order these errors per EIP / tx type? :smile:

Reply:
good point, will do